pkg/handlers/v1/nexposeassetproducer.go

```diff
@@ -65,4 +69,5 @@ func (h *NexposeScannedAssetProducer) Handle(ctx context.Context, in ScanInfo) {
 		}
 	}
 	wg.Wait()
+	stater.Count(fmt.Sprintf("totalassetsproduced.site:%s", in.SiteID), totalAssetsProduced)
```

Comment:
task: Metric names should not be templates and should not contain variable data. This creates a problem because it results in an large number of unique metric names. This both makes working with the data extremely difficult because most modern systems don't perform aggregations based on wildcards of metric names and eventually breaks most metrics systems over time by exceeding the maximum metric name capacity. Instead, this value should be added as a tag. However, we should only add this as a tag if the number of possible values is bounded to a small group. If this could be one of thousands then it should not be part of the metric as that also breaks most metric systems by exceeding the tag cardinality capacity.

Reply:
Yes! I went ahead and changed it to be a tag, didn't know that there was an optional tag parameter, my fault. The possible number of tags is fairly small, as it pertains to each site